Gemfile

```diff
@@ -19,6 +19,9 @@ group :test do
   gem 'zabbixapi',                   :require => false
 end
 
+gem 'rspec-puppet-facts', :git => 'https://github.com/ekohl/rspec-puppet-facts', :branch => 'find-facter-version'
+gem 'facterdb', :git => 'https://github.com/camptocamp/facterdb', :branch => 'master'
```

Comment:
I'd prefer not to rely on git branches unless really needed. Perhaps link to the commit/pr that we need to be released?

Reply:
1.3.0 was released so this can be removed.